lib/getDiffInMonths.ts

```diff
@@ -1,18 +1,29 @@
 import { ChronosDate, ensureDate } from './helpers/ensureDate';
+import { getDiffInCalendarMonths } from '.';
```

Comment:
Let's import the function from the file of that function, not from the index file?

This way it will be probably easier for bundler to tree-share everything.

Plus it won't be necessary to disable ESLint in index.ts.